src/identity-providers/add/DescriptorSettings.tsx

```diff
@@ -45,6 +45,12 @@ const Fields = ({ readOnly }: DescriptorSettingsProps) => {
     name: "config.validateSignature",
   });
 
+  const principalType = useWatch({
+    control: control,
```

Comment:
If the value has the same name as the key the part after the colon can be omitted.
```suggestion
    control,
```

Reply:
fixed, always forget this one - thanks

Reply:
Yeah, might be worth adding a linting rule for it.